src/main/webapp/app/complaints/complaint-response.service.ts

```diff
@@ -27,16 +73,24 @@ export class ComplaintResponseService {
             .map((res: EntityResponseType) => this.convertDateFromServer(res));
     }
 
-    private convertDateFromClient(complaintResponse: ComplaintResponse): ComplaintResponse {
+    public convertDateFromClient(complaintResponse: ComplaintResponse): ComplaintResponse {
```

Comment:
Why does it need to be public here? Isn't the only place this is used line 64 in this file?

Reply:
Yes but the convert functions should be public imo. So that you can easily use them in other services with entitites that reference this entity (for example complaints are connected to complaint responses etc)